MonkeyBot/Initializer.cs

```diff
@@ -296,6 +296,17 @@ private static async Task DiscordClient_GuildCreated(DiscordClient client, Guild
             _discordClient.Logger.LogInformation($"Joined guild {e.Guild.Name}");
             // Make sure to create the config;
             await _guildService.GetOrCreateConfigAsync(e.Guild.Id);
+            
+            var msgBuilder = new DiscordEmbedBuilder()
+                .WithTitle("Hello, I am ***Monkey Bot***")
+                .WithThumbnail("https://raw.githubusercontent.com/MarkusKgit/MonkeyBot/main/Logos/MonkeyBot.png",50,50)
+                .WithColor(DiscordColor.SpringGreen)
+                .AddField("Documentation","https://github.com/MarkusKgit/MonkeyBot")
+                .AddField("Discord server", "https://discord.gg/u43XvME")
+                .WithDescription("Use command `!help` to check what can I do for you.");
+            
+            var channel = e.Guild.GetDefaultChannel() ?? await e.Guild.CreateTextChannelAsync("Welcome channel");
```

Comment:
I decided to create some default channel when guild doesn't have any, but I am not sure if this is desired approach.

Reply:
Looks good to me, should in theory never happen anyway